Atm.py

```diff
@@ -0,0 +1,134 @@
+from Costumer import Costumer
+from pip._vendor.distlib.compat import raw_input
+
+
+class Atm:
+    def __init__(self, file_path):
+        self.costumers = {}
+        self.consumer = ''
+        self.need_update = False
+        self.file_path = file_path
+        self.on = True
+
+    def start_action(self):
+        self.update_costumers()
+        self.get_consumer()
+        while self.on:
+
+            if self.check_password():
+                self.get_desired_operation()
+                print('Thanks you for using our services!')
+                self.get_consumer()
+        if self.need_update:
+            self.update_details()
+
+    def update_costumers(self):
+        consumer_id = 0
+        password = 1
+        balance = 2
+        with open(self.file_path, 'r') as file:
+            costumers = [i[:i.find('\n')] for i in file.readlines()]
+        costumers = [i.split(',') for i in costumers]
+        for i in costumers:
+            if i[consumer_id] and i[password] and i[balance]:
+                costumer = Costumer(i[consumer_id], i[password], i[balance])
+                _id = i[consumer_id]
+                self.costumers[_id] = costumer
+        print(self.costumers)
+
+    def get_consumer(self):
+        consumer_id = raw_input('To turn off insert -1\nPlease enter your id:\n')
+        if int(consumer_id) != -1:
+            try:
+                self.consumer = self.costumers[consumer_id.strip()]
+            except:
+                print("No account for the inserted id, please try again")
+                self.get_consumer()
+        else:
+            self.on = False
+
+    def check_password(self):

```

Comment:
if all the passwords were bad you don't return anything, it would work with your if in start action function but it's a bad habit. please return something in all cases